pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -188,6 +274,20 @@ def forward(self, *inputs: torch.Tensor) -> torch.Tensor:
 
 
 class BranchBlock(nn.Module):
+    r"""BranchBlock from :cite:`ULVL2016`.
+
+    Joins the branch from the previous pyramid with the current level using a
+    :class:`~pystiche_paper.ulyanov_et_al_2016._modules.JoinBlock`.
+
+    Args:
+        deep_branch: Input from the previous pyramid.
+        shallow_branch: Input from the current level.
```

Comment:
The branches are modules and not images.

```suggestion
        deep_branch: Previous pyramid level.
        shallow_branch: Current pyramid level.
```